instructions.md

```diff
@@ -0,0 +1,89 @@
+# Installing LGTM Enterprise on Google Cloud
+## Preparation
+Throughout this setup guide a few variables are used that are useful to set in advance. Set the Google Cloud project you want to deploy to, the zone to create any zonal resources in and a unique name for this deployment. If you are upgrading an existing installation, set these values to the same values used when LGTM was first set up.
+
+```console
+project="my-project"
+deployment="lgtm"
+zone="us-central1-a"
+```
+
+## Controller
+### Creating a New LGTM Enterprise Controller
+A new LGTM Enterprise controller machine can be created using the `controller.py` template.
+```console
+gcloud deployment-manager deployments create --project "$project" --template=controller.py "$deployment" \
+	--properties "zone:$zone,administrator-email:email@example.com,administrator-password:MySuperSecretPassword"
+```
+
+The `--properties` flag can be used to provide a variety of options to customize your deployment:
+* `zone` - The Google Cloud zone to create resources in.
+* `virtual-machine-size` - The type of virtual machine to use.
```

Comment:
I think we need to give some information here (or refer them to the _LGTM Enterprise Installation and Upgrade Guide_ and give some details there - URL: https://help.semmle.com/lgtm-enterprise/ops/lgtm-enterprise-LATEST-installation-guide.pdf)

Reply:
You mean the system requirements document?

Reply:
The Sys Requirements doc doesn't give any specific advice for image-based deployments. I was thinking more of adding information about which machine to choose like we did for AMI installs. See Appendix B of the Install Guide: https://help.semmle.com/lgtm-enterprise/ops/lgtm-enterprise-LATEST-installation-guide.pdf

Though maybe a lot more lightweight than that information.

Reply:
Thanks, I'll update the link. I thought the CPU and memory requirements might be enough to allow people to select a machine type.